src/django-app/vehicles/policies.py

```diff
@@ -50,7 +50,7 @@ def scope_queryset(cls, request, qs):
             elif request.user.type == "employee":
                 employee_shop = EmployeeData.objects.get(user=request.user).shop
                 quote_requests = QuoteRequest.objects.filter(shop=employee_shop)
-            return qs.filter(pk__in=quote_requests.values_list("vehicle", flat=True))
+            return qs.filter(pk__in=quote_requests.values_list("vehicle", flat=True)) | qs.filter(customer=None)
```

Comment:
@tevisdoe This would likely not give the correct output. You need to do something like:
`return qs.filter(Q(pk__in=quote_requests.values_list("vehicle", flat=True)) | Q(customer=None))`

You need to import Q for this: `from django.db.models import Q`

Reply:
No, it's fine, you don't need to do that. I have tried both and both work. It simply combines the results of both queries in the above.

Reply:
I see, it's fine then. I'll approve it.